tests/api_fastapi/execution_api/routes/test_variables.py

```diff
@@ -75,3 +86,60 @@ def test_variable_get_access_denied(self, client):
                 "message": "Task does not have access to variable key1",
             }
         }
+
+
+class TestPostVariable(TestVariableEndpoint):
+    def test_should_create_variable(self, client, session):
+        payload = {"key": "var_create", "value": "{}"}
+        response = client.post(
+            "/execution/variables",
+            json=payload,
+        )
+        assert response.status_code == 200
+        response = client.get(f"/execution/variables/{payload['key']}")
```

Comment:
This is implicitly testing the `get` resposne too. Should we perhaps instead go directly to the DB and test what is in there?

Reply:
Yeah that makes sense. This is testing another API

Reply:
Handled the change